example/japanese-english-billingual-corpus.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/ruby
+
+require 'datasets'
+
+#WIP
+billingualCorpus = Datasets::JapaneseEnglishBilingualCorpus.new
+
+#emptyArticle = Datasets::JapaneseEnglishBilingualCorpus::Article.new
+#emptyTitle = Datasets::JapaneseEnglishBilingualCorpus::Title.new
+#emptySection = Datasets::JapaneseEnglishBilingualCorpus::Section.new
+#emptyPage = Datasets::JapaneseEnglishBilingualCorpus::Page.new
+emptySentence = Datasets::JapaneseEnglishBilingualCorpus::Sentence.new
+
+#articles = billingualCorpus.select{|x| x.is_a?(emptyArticle.class)}
+#titles = billingualCorpus.select{|x| x.is_a?(emptyTitle.class)}
+#sections = billingualCorpus.select{|x| x.is_a?(emptySection.class)}
+#pages = billingualCorpus.select{|x| x.is_a?(emptyPage.class)}
+sentences = billingualCorpus.select{|x| x.is_a?(emptySentence.class)}
```

Comment:
How about yielding only `Datasets::JapaneseEnglishBilingualCorpus::Article` instead of yielding all articles, titles, sections, pages and sentences like `Datasets::Wikipedia`? `Datasets::Wikipedia` yields only `Datasets::Wikipedia::Page` and it refers `Datasets::Wikipedia::Revision`.

```ruby
bilingual_corpus.each do |article|
  p article.title
  article.contents.each do |content|
    # content may be title or sentence
    if content.title?
      # title
      p [content.section.id, content.japanese, content.english]
    else # elsif content.sentence?
      # sentence
      p [content.section&.id, content.paragraph.id, content.id, content.japanese, content.english]
    end
  end
end
````

Reply:
Thank you!
I understood your idea and I think it would make it easier to use, too.
I also agree with you about the comments below.

Please push your changes if you can.